bitnami/redis/templates/_helpers.tpl

```diff
@@ -71,6 +71,15 @@ Return the appropriate apiGroup for PodSecurityPolicy.
 {{- end -}}
 {{- end -}}
 
+{{/*
+ Check PodSecurityPolicy create.
+ */}}
+ {{- define "podSecurityPolicy.checkCreate" -}}
+ {{- if and (eq .Values.podSecurityPolicy.create true) (eq .Values.podSecurityPolicy.enabled false) }}
+ {{- fail "In order to create PodSecurityPolicy, you also need to enable podSecurityPolicy.enabled field" }}
+ {{- end }}
+ {{- end }}
```

Comment:
We have a standard way to add values validation. Check this line: https://github.com/bitnami/charts/blob/0ade812deb74acdf1b9c79c52bfec735cd285f6c/bitnami/redis/templates/_helpers.tpl#L189

In short, we call all the validations in the same function and them show the errors in the NOTES.txt. Could you adapt the validation to this standard?

Reply:
Yes, I did the update. Please let me know if that's fine for you.